testpmd-lb-operator/config/manager/manager.yaml

```diff
@@ -31,6 +31,13 @@ spec:
             - name: ANSIBLE_GATHERING
               value: explicit
           image: controller:latest
+          lifecycle:
+            postStart:
+              exec:
+                command: ["/bin/sh", "-c", "echo Hello from the postStart handler"]
+            preStop:
+              exec:
+                command: ["/bin/sh", "-c", "echo Hello from the preStop handler"]
```

Comment:
are the requirements to have logs or to simply have lifecycle/postStart? 

```
        lifecycle:
          postStart: {}
          preStop: {}
```

Reply:
Good observation. Probably, according to the [tnf code](https://github.com/test-network-function/cnf-certification-test/blob/main/cnf-certification-test/lifecycle/suite.go#L259-L290), the test can be passed by just defining an empty postStart and preStop block, but I believe it's better to have an example of postStart and preStop to demonstrate how it works, even if it's just an `echo`.

Reply:
I think we're good then, let's merge